modules/airgap/setup/get_artifacts.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+## Uncomment the following lines to enable debug mode
+# set -x
+# echo "$@"
+
+# Usage: ./get_artifacts.sh k3s v1.27.5+k3s1
+# Usage: ./get_artifacts.sh rke2 v1.27.5+rke2 amd64 "flags" tar.gz
+
+product=$1
+version=$2
+arch=$3
+flags=${4}
+tarball_type=$5
+prodbin=$product
+
+
+check_arch(){
+  if [[ -n "$arch" ]] && [[ "$arch" = *"arm"* ]]; then
+    if [[ "$product" == "k3s" ]]; then
+      prodbin="k3s-arm64"
+    else
+      arch="arm64"
+    fi
+  else
+    arch="amd64"
+  fi
+}
+
+get_assets() {
+  echo "Downloading $product dependencies..."
+  if [[ "$product" == "k3s" ]]; then
+    url="https://github.com/k3s-io/k3s/releases/download/$version"
+    wget "$url"/k3s-images.txt
+    wget -O k3s-install.sh https://get.k3s.io/
+    wget -O k3s "$url"/"$prodbin"
+    if [ -n "$tarball_type" ]; then
+      wget "$url"/k3s-airgap-images-"$arch"."$tarball_type"
+    fi
+  elif [[ "$product" == "rke2" ]]; then
+    url="https://github.com/rancher/rke2/releases/download/$version"
+    wget "$url"/sha256sum-"$arch".txt
+    wget "$url"/rke2-images.linux-"$arch".txt
+    wget "$url"/rke2.linux-"$arch".tar.gz
+    wget -O rke2-install.sh https://get.rke2.io/
+    if [ -n "$tarball_type" ]; then
+      wget "$url"/rke2-images.linux-"$arch"."$tarball_type"
+    fi
+  else
+    echo "Invalid product: $product. Please provide k3s or rke2 as product"
+  fi
+}
+
+get_cni_assets() {
+  if [[ -n "$flags" ]] && [[ "$flags" =~ "cni" ]]; then
+    url="https://github.com/rancher/rke2/releases/download/$version"
+    cnis=("calico" "canal" "cilium" "flannel" "multus")
+    for cni in "${cnis[@]}"; do
+      if [[ "$flags" =~ $cni ]]; then
```

Comment:
If cni is not passed in server flags we need to default it to canal

Reply:
I think it already default to canal if no server flags is empty, will check again and confirm

Reply:
Confirmed default cni images are part of the images.txt